server/config/services.yaml

```diff
@@ -40,4 +40,6 @@ services:
     App\Security\OpenApi\LoginDecorator:
         decorates: 'api_platform.openapi.factory'
         arguments: ['@.inner']
-        decoration_priority: 0
\ No newline at end of file
+        decoration_priority: 0
+
+    Symfony\Bridge\Doctrine\Security\User\UserLoaderInterface: '@App\Security\Doctrine\Repository\UserRepository'
```

Comment:
T'es sur que c'est obligatoire, il ne suffit pas d'ajouter l'interface au repo ?

Reply:
sans, j'ai cette erreur-ci: `Symfony\Component\DependencyInjection\Exception\RuntimeException : Cannot autowire service "App\Security\UseCase\RequestForgottenPassword\RequestForgottenPasswordHandler": argument "$userLoader" of method "__construct()" references interface "Symfony\Bridge\Doctrine\Security\User\UserLoaderInterface" but no such service exists. You should maybe alias this interface to the existing "App\Security\Doctrine\Repository\UserRepository" service.`

Reply:
Je fais ceci comme DI dans le Handler
```php
    public function __construct(
        private UserGateway $userGateway,
        private UserLoaderInterface $userLoader,
        private UuidV6Factory $uuidFactory,
        private MailerInterface $mailer
    )
```
Si je n'injecte que le `UserGateway`, je n'ai pas accès à la méthode `loadUserByIdentifier`...

Reply:
En fait, je vois pas l'intérêt de passer par UserLoaderInterface.

Reply:
oui tout à fait... juste alors quel est l'intérêt d'implémenter l'interface `UserLoaderInterface` pour le UserRepository, c'est utilisé de manière auto par APIP?

Reply:
Mon idée était de ne pas en rajouter et réutiliser ce qui existait... mais je dois de toutes façons en rajouter une pour getByToken... une de plus ou une de moins ;-)

Reply:
corrigé dans le dernier commit